MdeModulePkg/Library/PlatformBootManagerLibNull/PlatformBootManager.c

```diff
@@ -74,3 +74,66 @@ PlatformBootManagerUnableToBoot (
 {
   return;
 }
+
+/**  MU_CHANGE [BEGIN]
+   Do Platform specific action required at start of BDS
+
+**/
+VOID
+EFIAPI
+PlatformBootManagerBdsEntry (
+  VOID
+  )
+{
+  return;
+}
+
+/**
+ProcessBootCompletion
+*/
+VOID
+EFIAPI
+PlatformBootManagerProcessBootCompletion (
+  IN EFI_BOOT_MANAGER_LOAD_OPTION  *BootOption
+  )
+{
+  return;
+}
+
+/**
+ HardKeyBoot
+*/
+VOID
+EFIAPI
+PlatformBootManagerPriorityBoot (
+  UINT16  **BootNext
+  )
+{
+  return;
+}
+
+/**
+ BDS Deadloop - error, unable to boot any boot option
+*/
+VOID
+EFIAPI
+PlatformBootManagerDeadloop (
+  VOID
+  )
+{
+  return;
+}
+
+/**
+  OnDemandConInCOnnect
+ */
+VOID
+EFIAPI
+PlatformBootManagerOnDemandConInConnect (
+  VOID
+  )
+{
+  return;
+}

```

Comment:
I thought @kuqin12 had at least part of this already in PR form?

Reply:
If that's true I'm happy to give @kuqin12 this commit just so it can go with like changes.

Reply:
That was for the content in silicon ARM. This change is still needed.